content/2. Ondersteunende-informatie/3. Processen/3. Ontwerpproces/1. Opstellen ontwerp/Schematechnieken/1. Klassendiagram.md

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
Voor de juiste scheiding van de content moet elk onderwerp in een eigen folder zitten en dan pas een md file. Voor de scheiding is het handig om "Schematechnieken" te hernoemen naar "Klassendiagram" of in "Schematechnieken" een folder te maken "Klassendiagram" en in "Klassendiagram" het md document "1. Klassendiagram" neer te zetten.